features/support/before-hook.js

```diff
@@ -30,6 +31,18 @@ function promiseToWait(timeToWait) {
 }
 
 /**
+ * Generate a random string
+ * @method randomBytes
+ * @param {Number}      size    Number of bytes to generate
+ * @return {Promise}
+ */
+function randomBytes(size) {
```

Comment:
In terms of dependencies, `crypto` is part of node. It doesn't have to be in the `package.json` or anything, it's just there.

If all of the tests are run in sequence then setting the description to be a literal value should work, however the setup `Given "calvin" owns an existing API token named "tiger"` doesn't actually set the description to anything, it just makes sure the token exists. So if the `update` test is called repeatedly without a `create` or `delete` between it, it'd keep setting the description to the same value.

That could be fixed either by making an `update` request in the setup step if the token is found to already be there, or making a `get` request as part of this step and then calling update with a value that's different than the one already there (like alternating between "tiger" and "not tiger" or something). I guess it depends what you think would be cleaner.